src/motion_click_app.c

```diff
@@ -56,21 +56,15 @@ static void print_usage(const char *program)
         program);
 }
 
-static int parse_cmd_opts(int argc, char *argv[], uint8_t *gpio)
+static int8_t parse_cmd_opts(int argc, char *argv[], uint8_t *gpio)
```

Comment:
why did you change this? Sized types are best used only where there is a good reason to do so (interface requirements, hardware width matching, alignment, optimisation). Otherwise it's best to use just standard types as the size is completely irrelevant here.